lib/url-template.js

```diff
@@ -13,16 +13,36 @@
   function UrlTemplate() {
   }
 
+  var percentTwenties = /%20/g;
+
+  /**
+   * @private
+   * @param {string} str
+   * @return {string}
+   */
+  UrlTemplate.prototype.usePlusSignSpacing = function (str) {
+    if (percentTwenties.test(str)) {
```

Comment:
nit. is this needed? Presumably if there were not any instances of this, replace would not effect the string.

Reply:
Eh. seems to be a pattern used throughout, so alrighty.